dist/ntrip-catalog.json

```diff
@@ -0,0 +1,270 @@
+{
+    "$schema": "https://ntrip-catalog.org/schemas/v0.1/ntrip-catalog.schema.json",
+    "release": 0,
```

Comment:
Maybe add a `"comment"` that says: "This file has been automatically generated by blah, blah on that day"

Reply:
OK, don't include the date. In any case, checking JSON for equivalence is difficult in itself. If you really want to facilitate that, it has to be canonicalized.